src/flask/helpers.py

```diff
@@ -1122,3 +1123,45 @@ def is_ip(value):
             return True
 
     return False
+
+
+def run_async(func):
+    """Return a sync function that will run the coroutine function *func*."""
+    try:
+        from asgiref.sync import async_to_sync
+    except ImportError:
+        raise RuntimeError(
+            "Install Flask with the 'async' extra in order to use async views."
+        )
+
+    @wraps(func)
+    def outer(*args, **kwargs):
+        """This function grabs the current context for the inner function.
+
+        This is similar to the copy_current_xxx_context functions in the
+        ctx module, except it has an async inner.
+        """
+        ctx = None
+        if _request_ctx_stack.top is not None:
+            ctx = _request_ctx_stack.top.copy()
+        elif _app_ctx_stack.top is not None:
+            ctx = _app_ctx_stack.top.copy()
```

Comment:
`AppContext` doesn't have a `copy` method.

Reply:
Ah, it does in Quart... fixed.